### PR TITLE
Details

### PR DESCRIPTION
**Pull request by Cocatrix**
HMI view.
Buttons are clickable, left : methods for calling, texting...

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ import { AuthProvider } from '../providers/auth/auth';
 import { ErrorProvider } from '../providers/error/error';
 import { HttpClientModule } from '@angular/common/http';
 import { IonicStorageModule } from '@ionic/storage';
+import {PhonePipe} from "../pipes/phone/phone";
```
Comment (Nilmanduil) [inline src/app/app.module.ts]: TSLint error

File: src/pages/details-contact/details-contact.module.ts [hunk 2]
```diff
@@ -1,13 +1,16 @@
 import { NgModule } from '@angular/core';
 import { IonicPageModule } from 'ionic-angular';
 import { DetailsContactPage } from './details-contact';
+import Contact from '../../models/Contact';
+import {PhonePipe} from "../../pipes/phone/phone";
```
Comment (Nilmanduil) [inline src/pages/details-contact/details-contact.module.ts]: TSLint error

File: src/pages/login/login.ts [hunk 3]
```diff
@@ -31,6 +31,7 @@ export class LoginPage {
   }
 
   ionViewDidLoad() {
+    this.viewCtrl.dismiss().then(); // TODO - Remove (useful for test only)
```
Comment (Nilmanduil) [inline src/pages/login/login.ts]: Dismiss immediately after loading ?

Comment (Cocatrix) [reply src/pages/login/login.ts]: It is for testing without typing phone and password every time.
You're right, I will comment it for the master branch.

**Pull request merged**